app/controllers/api/v1/customers_random_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::CustomersRandomController < ApiBaseController
+  respond_to :json, :xml
+
+  def show
+    respond_with Customer.order("RANDOM()").first
```

Comment:
I was not aware of this SQL query! I'll be adding it to the missing methods in my models I have as well.